src/components/Crews/ResultOverview.vue

```diff
@@ -113,6 +136,18 @@ export default {
                     thClass: 'font-italic',
                     class: 'text-center d-none d-sm-table-cell'
                 },
+                ...findClockingLocations(this.crews).map(location => ({
```

Comment:
Is de club en zoek overzicht ook bijgewerkt?

Reply:
Willen we dat? aangezien daar meerdere velden door elkaar staan dus tussentijden vergelijken misschien niet echt nodig/nuttig is.

Reply:
Ik denk het wel omdat tussen tijden vaak gelijkt zijn over de volledige westrijd.